recipes/plothist_utils/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "plothist_utils" %}
+{% set version = "0.0.1" %}
+{% set python_min = "3.9" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default.

```suggestion
```